UserService/app/src/main/java/com/bookmyshow/userservice/dto/RoleRequestDTO.java

```diff
@@ -0,0 +1,37 @@
+package com.bookmyshow.userservice.dto;
+
+import javax.validation.constraints.NotBlank;
+import java.io.Serializable;
+
+public class RoleRequestDTO implements Serializable {
+
+    @NotBlank(message = "Role must not be empty")
+    private String role;
+
+    private String description;
+
+    public RoleRequestDTO(String role, String description) {
+        this.role = role;
+        this.description = description;
+    }
```

Comment:
How about we have only one DTO for role than having request and response, since fields are same.